### ISSUE TITLE
Screen broken

### ISSUE THREAD

**Issue by Daniel99j2** [maruohon/malilib#113]
If the GUI scale is more than 4, things overlap and go off the screen
(PS: i think it's this mod that adds this style config, tell me if i'm wrong)
<img width="1248" alt="Screenshot_20230208_063853" src="https://user-images.githubusercontent.com/121533735/217506980-4d2ca2dd-14ba-4506-84e9-0f050eea3bfd.png">

**Comment by maruohon** [comment]
I *might* add a horizontal scroll bar at some point, but other than that there isn't anything that I can "fix" if the user uses a screen scale such that "only one button fits on screen".

However the development version of malilib also has an option to use a different GUI scale for just all the malilib-based screens. That way you can use a larger scale for vanilla and other mods, but then use a smaller scale for any malilib-based screens, which are designed for a 1280x720 resolution using scale 2 (so basically 640x360 usable resolution for the screen components/widgets). Anything that results in a significantly smaller effective/usable resolution is just going to look bad no matter what.

So basically this is the size and scale that things are "meant to look like":
![2023-02-01_08 12 50](https://user-images.githubusercontent.com/2331050/217643130-51f1ca5a-c4a4-4f29-b505-e95978041261.png)
